src/main/java/org/tikv/common/TiSession.java

```diff
@@ -154,16 +174,20 @@ public TiSession(TiConfiguration conf) {
               conf.getIdleTimeout());
     }
 
-    this.client = PDClient.createRaw(conf, channelFactory);
+    this.client = PDClient.createRaw(conf, keyCodec, channelFactory);
     this.enableGrpcForward = conf.getEnableGrpcForward();
     if (this.enableGrpcForward) {
       logger.info("enable grpc forward for high available");
     }
     if (conf.isWarmUpEnable() && conf.isRawKVMode()) {
-      warmUp();
+      warmup();
```

Comment:
Could we keep the camel style? In TiConfiguration, the variables of warm-up are all camel style.